wfsim/load_resource.py

```diff
@@ -276,13 +281,15 @@ def __init__(self, config=None):
 
             # Garfield luminescence timing samples
             # if config.get('s2_luminescence_model', False) == 'garfield':
-            if 'garfield' in config.get('s2_luminescence_model', ''):
-                s2_luminescence_map = straxen.get_resource(files['s2_luminescence'], fmt='npy_pickle')['arr_0']
-                # Get directly the map for the simulated level
-                liquid_level_available = np.unique(s2_luminescence_map['ll'])  # available levels (cm)
-                liquid_level = config['gate_to_anode_distance'] - config['elr_gas_gap_length']  # cm
-                liquid_level = min(liquid_level_available, key=lambda x: abs(x - liquid_level))
-                self.s2_luminescence = s2_luminescence_map[s2_luminescence_map['ll'] == liquid_level]
+            if 'garfield_gas_gap' in config.get('s2_luminescence_model', ''):
+                s2_luminescence_map = straxen.get_resource(files['s2_luminescence_gg'], fmt='npy')
+                self.s2_luminescence = s2_luminescence_map
+                self.garfield_gas_gap_map = make_map(files['garfield_gas_gap_map'], fmt = 'json')
+
+            elif 'garfield' in config.get('s2_luminescence_model', ''):
+                s2_luminescence_map = straxen.get_resource(files['s2_luminescence'], fmt='npy')
```

Comment:
You change from `fmt='npy_pickle'` to `fmt='npy'`, does it work anyway?

Reply:
It works when we test it locally